src/clojuratica/core.clj

```diff
@@ -34,88 +34,110 @@
 ; ***** END LICENSE BLOCK *****
 
 (ns clojuratica.core
```

Comment:
it's worth understanding this namespace as this is where the connection with the WL is initiated

Also NOTE: loading this ns is side-effecting!!! (see below)